sway-core/Cargo.toml

```
@@ -20,6 +20,7 @@ fuel-asm = "0.2"
 fuel-crypto = "0.4.0"
 fuel-vm = "0.5"
 hex = { version = "0.4", optional = true }
+itertools = "0.10.3"
```

Is there a reason not to use `0.10`?
```suggestion
itertools = "0.10"
```

---

No reason, I just used what was shown on crates.io.